app/routes/user/notifications/index.tsx

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+
+import type { DataFunctionArgs } from '@remix-run/node'
+import { redirect } from '@remix-run/node'
+import { Form, Link, useLoaderData } from '@remix-run/react'
+import { Button, ButtonGroup, Grid, Icon } from '@trussworks/react-uswds'
+import SegmentedCards from '~/components/SegmentedCards'
+import TimeAgo from '~/components/TimeAgo'
+import { getFormDataString } from '~/lib/utils'
+import { EmailNotificationVendingMachine } from '../email_notifications.server'
+
+function sendTestEmail(id?: string): void {
+  console.log(id)
```

Comment:
Whoops! Need to implement this.

Reply:
Do we want this to just send an email with nodemailer or have this trigger something through SES?

Reply:
Let's use the SES API.